crates/sui-indexer/src/store/pg_indexer_store.rs

```diff
@@ -11,17 +11,19 @@ use std::time::Instant;
 use async_trait::async_trait;
 use core::result::Result::Ok;
 use diesel::dsl::max;
+use diesel::r2d2::R2D2Connection;
 use diesel::upsert::excluded;
 use diesel::ExpressionMethods;
 use diesel::OptionalExtension;
 use diesel::{QueryDsl, RunQueryDsl};
+use downcast::Any;
```

Comment:
is this used?

Reply:
Yes, this gets used in the macro but i'll double check (and remove if not)